cfme/tests/networks/nuage/test_automation.py

```diff
@@ -12,3 +22,52 @@ def test_auth_attributes_of_nuage_ae_class(appliance):
     schema_fields = nuage_class.schema.schema_field_names
 
     assert all(expected in schema_fields for expected in expected_fields)
+
+
+@pytest.mark.parametrize('copy_ae_instance_to_domain', ['ManageIQ/System/Event/EmsEvent/Nuage'
+                                                        '/nuage_enterprise_create'], indirect=True)
+def test_nuage_event_playbook_triggers(ansible_repository, copy_ae_instance_to_domain,
+                                       networks_provider):
+    """
+    We copy AE instance (nuage_enterprise_create) on path `ManageIQ/System/Event/EmsEvent/Nuage`
+    and copy it to newly created domain. Then we create new AE method and attach it to previously
+    copied AE instance. This method is executed when Nuage Enterprise is created. Method runs
+    `printout.yaml` ansible playbook and display information about Nuage fields.
+    With the help of `LogValidator` we then check for appropriate line to appear in the `evm.log`.
+    """
+    nuage_ae_class = copy_ae_instance_to_domain.klass
+    ae_method = nuage_ae_class.methods.create(name="printout", location='playbook',
+                                              repository=ansible_repository.name,
+                                              playbook='printout.yaml',
+                                              machine_credential="CFME Default Credential",
+                                              logging_output='Always')
+    username = random_name()
+    with update(copy_ae_instance_to_domain):
+        copy_ae_instance_to_domain.fields = {
+            'nuage_username': {'value': username},
```

Comment:
There is actually a problem with password field, because password field has a different `id` value (`id='inst_password_value'`) inside HTML tag then the rest of the fields (`id='inst_value')`. File `cfme/automate/explorer/instance.py` line 117 and 124.

\cc @miha-plesko

Reply:
Yaaay. Okay, I see you managed to resolve this by simply not including the password, which works with me 👍